content/en/docs/setup/best-practices/certificates.md

```diff
@@ -22,6 +22,7 @@ This page explains the certificates that your cluster requires.
 Kubernetes requires PKI for the following operations:
 
 * Client certificates for the kubelet to authenticate to the API server
+* Server certificates for the [kubelet endpoint](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#client-and-serving-certificates)
```

Comment:
```suggestion
* Kubelet [server certificates](docs/reference/command-line-tools-reference/kubelet-tls-bootstrapping/#client-and-serving-certificates)
  for the the API server to talk to the kubelets
```

Reply:
👋 @dialogbox .
Would you update the link as suggested? Thanks!